packages/ui/src/domainModules/collectionMammals/components/CatalogNumberModal/index.js

```diff
@@ -70,7 +70,7 @@ class CatalogNumberModal extends PureComponent {
 
     return (
       <React.Fragment>
-        <Modal open size="small">
+        <Modal className="ui form" open size="small">
```

Comment:
since there are many form modals and there might be more, maybe you can make a FormModal component that you put in the form module that just renders something like this
```
const { children, ...rest } = this.props

<Modal className="ui form" {...rest} />
  {children}
</Modal>

and then use it instead. maybe that would decrease the risk of forgetting to add ui form. what do you think?